Resources/Prototypes/_SV/Entities/Clothing/Head/hats.yml

```
@@ -45,3 +46,31 @@
     sprite: _SV/Clothing/Head/Hats/admiral_cap.rsi
   - type: Clothing
     sprite: _SV/Clothing/Head/Hats/admiral_cap.rsi
+
+- type: entity
+  parent: [ClothingHeadBase]
+  id: ClothingHeadHatTricornCaptain
+  name: captain's tricorn
+  description: A captain hat adorned by multiple colourful feathers.
+  components:
+  - type: Sprite
+    sprite: _SV/Clothing/Head/Hats/captain_tricorn.rsi
+  - type: Clothing
+    sprite: _SV/Clothing/Head/Hats/captain_tricorn.rsi
+    tags:
+    - PetWearable
+    - CorgiWearable
+    - ClothMade
+    - Recyclable
+    - WhitelistChameleon
+
+- type: entity
+  parent: [ClothingHeadBase, ClothingHeadHatTricornCaptain]
```

```suggestion
  parent: ClothingHeadHatTricornCaptain
```
ClothingHeadBase is already inherited from ClothingHeadHatTricornCaptain. At the same time, Upstream does not inherit from different hats and just goes from ClothingHeadBase on all clothing